.github/workflows/ci.yml

```
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python_version: [3.10]
+        python_version: '3.10'
```

```suggestion
        python_version: ['3.10']
```

I think this still needs to be a list even though there's only one entry.